site/content/en/docs/formats/yolo_detection.md

```diff
@@ -0,0 +1,178 @@
+   │ # a list of non-format labels (optional)  # file with list of classes
+   ├── data.yaml    # file with dataset information
+   ├── train.txt    # list of image paths in train subset [Optional]
+   ├── valid.txt    # list of image paths in valid subset [Optional]
+   │
+   ├── images/
+   │   ├── train/  # directory with images for train subset
+   │   │    ├── image1.jpg
+   │   │    ├── image2.jpg
+   │   │    ├── image3.jpg
+   │   │    └── ...
+   │   ├── valid/  # directory with images for validation subset
+   │   │    ├── image11.jpg
+   │   │    ├── image12.jpg
+   │   │    ├── image13.jpg
+   │   │    └── ...
+   ├── labels/
+   │   ├── train/  # directory with annotations for train subset
+   │   │    ├── image1.txt
+   │   │    ├── image2.txt
+   │   │    ├── image3.txt
+   │   │    └── ...
+   │   ├── valid/  # directory with annotations for validation subset
+   │   │    ├── image11.txt
+   │   │    ├── image12.txt
+   │   │    ├── image13.txt
+   │   │    └── ...
+```
+
+`data.yaml` should have the following content, it is not necessary to have both subsets, but necessary to have one of them. Additionally, class numbers should be zero-indexed (start with 0).:
+
+```yaml
+# Train/val/test sets as 1) dir: path/to/imgs, 2) file: path/to/imgs.txt, or 3) list: [path/to/imgs1, path/to/imgs2, ..]
+path:  ./ # dataset root dir
+train: train.txt  # train images (relative to 'path') 4 images
+val: val.txt  # val images (relative to 'path') 4 images
+test:  # test images (optional)
+
+# Classes
+names:
+  0: person
+  1: bicycle
+  2: car
+  # ...
+  77: teddy bear
+  78: hair drier
+  79: toothbrush
+```
+
+Files `train.txt` and `valid.txt` should have the following structure:
+
+```txt
+<path/to/image1.jpg>
+<path/to/image2.jpg>
+...
+```
+
+Files in directories `labels/train/` and `labels/valid/` should contain information about labeled bounding boxes for images in `images/train` and `images/valid` respectively. If there are no objects in an image, no `.txt` file is required:
+
+```txt
+# labels/image1.txt:
+# <label_index> <x_center> <y_center> <width> <height>
+0 0.250000 0.400000 0.300000 0.400000
+3 0.600000 0.400000 0.400000 0.266667
+...
+```
+
+Box coordinates must be in normalized `xywh` format (from 0 to 1). If your boxes are in pixels, you should divide `x_center` and `width` by `image width`, and `y_center` and `height` by `image height`
+
+## Export to other formats
+
+Datumaro can convert a YOLO-Detection dataset into any other format Datumaro supports. For successful conversion, the output format should support the object detection task (e.g., Pascal VOC, COCO, TF Detection API, etc.).
+
+To convert a YOLO-Detection dataset to other formats, use the following commands:
+
+```bash
+datum create
+datum add -f yolo_detection <path/to/yolov8/>
+datum export -f coco_instances -o <output/dir>
+```
+or
+```bash
+datum convert -if yolo_detection -i <path/to/dataset> -f coco_instances -o <path/to/dataset>
```

Comment:
and with renaming default to one of the proposed names too, it shows like the command is accomplished, but the folder with results is empty